python/lsst/meas/deblender/baseline.py

```diff
@@ -300,7 +300,7 @@ def deblend(footprint, maskedImage, psf, psffwhm,
         # FIXME -- just within the bbox?
         stats = afwMath.makeStatistics(varimg, mask, afwMath.MEDIAN)
         sigma1 = math.sqrt(stats.getValue(afwMath.MEDIAN))
-        log.logdebug('Estimated sigma1 = %f' % sigma1)
+        log.trace('Estimated sigma1 = %f', sigma1)
```

Comment:
Why the change in level from debug to trace (here and above on line 276: `Log.TRACE`)